.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "tests/ethereum/tests"]
-	path = tests/ethereum/tests
+	path = pevm/tests/ethereum/tests
```

Comment:
We also need to update the `working-directory` for lint and tests in CI, which would actually fail currently.